src/wp-content/themes/twentysixteen/inc/back-compat.php

```diff
@@ -39,7 +39,7 @@ function twentysixteen_switch_theme() {
  */
 function twentysixteen_upgrade_notice() {
 	printf(
-		'<div class="error"><p>%s</p></div>',
+		'<div class="notice notice-error"><p>%s</p></div>',
```

Comment:
Don't change back-compat classes, as the context where these run may not include the necessary support.